checkoutmanager/runner.py

```diff
@@ -22,6 +22,8 @@
     'co': "Grab missing checkouts from the server",
     'missing': "Print directories that are missing from the config file",
     'out': "Show changesets you haven't pushed to the server yet",
+    'cu': "Check for updates that will be pulled in with 'up'",
```

Comment:
cu = check update, I assume? Or is it an abbreviation in one of the VCSs that we support?

Names need to be **good** and I'm not sold yet on "cu". Wouldn't `in` be a better one? It *is* more or less the reverse of the `out` command!